loader/src/main/kotlin/net/weavemc/loader/util/FileManager.kt

```diff
@@ -9,11 +9,14 @@ internal object FileManager {
     val MODS_DIRECTORY = getOrCreateDirectory("mods")
     val DUMP_DIRECTORY = getOrCreateDirectory(".bytecode.out")
 
+    private fun buildPath(vararg parts: String) =
+        parts.joinToString(File.separator)
+
```

Comment:
This exists already: [docs](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io.path/-path.html)

Reply:
That is not what I want at all, but I'll try and rewrite this to be cleaner